docs/guide/unspentoutput.md

```diff
@@ -0,0 +1,39 @@
+title: UnspentOutput
+description: A stateless model to represent an unspent output and associated information.
+---
+# UnspentOutput
+
+## Description
+
+`bitcore.Transaction.UnspentOutput` is a class with stateless instances that provides information about an unspent output:
+- Transaction ID and output index
+- The "scriptPubKey", the script included in the output
+- Amount of satoshis associated
+- Address, if available
+
+## Parameters
+
+The constructor is quite permissive with the input arguments. It can take outputs straight out of bitcoind's getunspent RPC call. Some of the names are not very informative for new users, so the UnspentOutput constructor also understands these aliases:
+- `scriptPubKey`: just `script` is also accepted
+- `amount`: expected value is in BTC, but also `satoshis` is accepted
```

Comment:
`satoshis` is a field, could be confused as accepting either value BTC or satoshis as input for `amount`